tests/ut/test_factory.py

```diff
@@ -30,7 +45,7 @@ def test_create_simple_cache():
 
 def test_create_cache_with_everything():
     redis = _create_cache(
-        RedisCache,
+        SimpleMemoryCache,
```

Comment:
I've replaced `RedisCache` with `SimpleMemoryCache` here since it seemed not to feature anything redis-specific.

Reply:
OK, variable should be renamed though.

Reply:
Right, fixed.